netbench/cdk/src/main/java/com/aws/ClientServerStack.java

```diff
@@ -0,0 +1,85 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.aws;
+
+import software.constructs.Construct;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.services.s3.Bucket;
+import software.amazon.awscdk.services.ecs.Cluster;
+import software.amazon.awscdk.services.ec2.Vpc;
+import software.amazon.awscdk.services.ecs.*;
```

Comment:
nit: Usually star imports should be avoided, but unfortunately that conflicts with the Jetbrains default settings

Reply:
Yeah, that's something that I had a placeholder since I was using a bunch of constructs from ecs, I will definitely fix that.